science-containers/Dockerfiles/firefly/Dockerfile

```diff
@@ -1,15 +1,16 @@
-FROM gradle:jdk21 AS builder
+FROM gradle:jdk21 AS base
 
 COPY . /firefly
 WORKDIR /firefly/cadc-sso
 
 SHELL ["/bin/bash", "-eo", "pipefail", "-c"]
 
+FROM base AS builder
 RUN set -eux; \
     gradle build --info \
     && ls -l lib/build/libs/cadc-sso-*.jar
```

Comment:
For me, the Docker build puts the built JAR in `build/libs/cadc-sso*.jar`.  Can you confirm that?